src/components/Button.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledButton = styled.button`
+  background: ${p => p.isGradient ? 'linear-gradient(92.58deg, #0DEFE1 0%, #78FF96 100%)' : p.backgroundColor};
+  color: ${p => p.textColor};  
+  height: ${p => p.height};
+  width: ${p => p.width};
+  border-radius: ${p => p.borderRadius};
+  border: none;
+
+  ${(p) =>
```

Comment:
Design team updated the Figma file with hover states for the buttons, could we change these to match? Looks like on hover the background turns transparent and the text + border take on the background color.
![image](https://user-images.githubusercontent.com/5078356/136853251-1fa00034-90d3-4cc0-a355-ea24e514874a.png)

Reply:
Sweet, thank you!